client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -212,7 +212,7 @@ const NotificationSubscriptions = createReactClass( {
 						</FormFieldset>
 
 						<FormFieldset>
-							<FormLegend>{ this.props.translate( 'Block Emails' ) }</FormLegend>
+							<FormLegend>{ this.props.translate( 'Block emails' ) }</FormLegend>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.